rc/lsp.kak

```diff
@@ -2057,7 +2044,7 @@ define-command lsp-enable -docstring "Default integration with kak-lsp" %{
     hook -group lsp global BufSetOption lsp_config=.* lsp-did-change-config
     hook -group lsp global BufSetOption lsp_server_configuration=.* lsp-did-change-config
     hook -group lsp global InsertIdle .* lsp-completion
-    hook -group lsp global InsertCompletionHide .* lsp-completion-dismissed
+    hook -group lsp global InsertCompletionHide .+ lsp-completion-accepted
```

Comment:
Is the completion always accepted on hide?

I guess it can be due to the change from `.*` to `.+`, where there is no range when empty, but it is not obvious to translate as cancel when reading the code.